$OpenBSD$

This makes GEnie generate correct CC and CXX values, overriding the
internal hardcoded values.

Index: scripts/toolchain.lua
--- scripts/toolchain.lua.orig
+++ scripts/toolchain.lua
@@ -173,6 +173,9 @@ function toolchain(_buildDir, _subDir)
 		end
 
 		if "openbsd" == _OPTIONS["gcc"] then
+			premake.gcc.cc  = "cc"
+			premake.gcc.cxx = "c++"
+			premake.gcc.ar  = "ar"
 			location (_buildDir .. "projects/" .. _subDir .. "/".. _ACTION .. "-openbsd")
 		end
 
